.github/labels.yml

```diff
@@ -0,0 +1,315 @@
+# define labels including kinds / areas etc
+kind:
```

Comment:
Can we delete some labels and areas not suitable for us?

Like:

- mesos-flake
- friction

in kind and lots of others in area.